packages/components/src/date-time/style.scss

```diff
@@ -118,51 +118,15 @@
 			color: $dark-gray-500;
 		}
 
-		.components-datetime__time-am-button {
-			margin-left: 8px;
-			margin-right: -1px;
-			border-radius: 3px 0 0 3px;
-		}
-
-		.components-datetime__time-pm-button {
-			margin-left: -1px;
-			border-radius: 0 3px 3px 0;
-		}
-
-		.components-datetime__time-am-button:focus,
-		.components-datetime__time-pm-button:focus {
-			position: relative;
-			z-index: 1;
-		}
-
-		.components-datetime__time-am-button.is-pressed,
-		.components-datetime__time-pm-button.is-pressed {
-			background: $light-gray-300;
-			border-color: $dark-gray-100;
-			box-shadow: inset 0 2px 5px -3px $dark-gray-500;
-			&:focus {
-				box-shadow:
-					inset 0 2px 5px -3px $dark-gray-500,
-					0 0 0 1px $white,
-					0 0 0 3px $blue-medium-focus;
-			}
-		}
-
```

Comment:
What's all this red about :)

Reply:
All that red duplicated the rules from the ButtonGroup component. In all my testing they are unnecessary with the refactor to use the actual ButtonGroup (and to toggle the selected button to primary in the ternery). You can recognize the margin-left: -1px from the button group, where it's used to ensure their borders stack. The z-index to ensure focus is uncropped is also there.

Reply:
The border radii for the am/pm buttons are even replaced by first-child/last-child rules in the buttongroup.